csharp_dotnetcore/18.Multi-Lingual-Bot/Translation/MicrosoftTranslator.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using MultiLingualBot.Translation.Model;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MultiLingualBot.Translation
+{
+    public interface ITranslator
+    {
```

Comment:
I'm not sure we need the interface for a sample. We're not planning to test/mock, so here it may just be adding complexity.